theme6/source/partials/_footer.erb

```diff
@@ -0,0 +1,40 @@
+<div class="contenedor">
+      <footer>         
+          <div class="contenedorsocial">
```

Comment:
When adding a name to a class, try and do it in English. I see that you have `boxsocialmedia` below, which is in English. So, in order to keep the consistency name all your classes in English. Also, it is good practice for your English skills.

Reply:
:fuckyeah: 🤓